tests/integration_tests/test_shell_hemisphere_mixed.py

```diff
@@ -199,3 +218,46 @@ def elem_call_back(
             )
 
         return tacs_probs, fea_assembler
+
+    def test_jacobian_scaling(self):
+        """Test that the Jacobian contribution from the first order centrifugal load elements is scaled correctly by the
+        loadScale parameter.
+
+        We test that J(loadscale=1) - J(loadscale=0) = 2*(J(loadscale=0.5) - J(loadscale=0))
+        """
+        for prob in self.tacs_probs:
+            if prob.name != "Centrifugal_firstOrder":
+                continue
+            prob._updateAssemblerVars()
+
+            prob.loadScale = 0.0
+            prob._factorOnNext = True
+            prob._initializeSolve()
```

Comment:
Just replace these lines (happens in a couple other tests) with `prob.setLoadScale(0.0)`.  Everything else looks good to me

Reply:
I actually wanted to keep the `prob.loadScale = value` form for one of these, to test that this method of setting the loadFactor works too.

Reply:
Setting `prob._factorOnNext = True` is necessary here, otherwise the stiffness matrix isn't updated during `_initializeSolve`

Reply:
Ah, I didn't realize you made `prob.loadScale` a setter function. I'm alright with it then. Can we move `_factorOnNext = True` into the `prob.setLoadScale` method then so the user doesn't have to update it?